File: models/exercises.js
module.exports = ( sequelize, Sequelize ) => {
    const Exercise = sequelize.define(
        "Exercise",
        {
            Name: {
                type: Sequelize.DataTypes.STRING,
                unique: true,
                allowNull: false,
            },
            Muscle: {
                type: Sequelize.DataTypes.STRING,
                unique: false,
                allowNull: false,
            },
            Equipment: {
                type: Sequelize.DataTypes.STRING,
                unique: false,
                allowNull: false,
            }
        },
        {
            timestamps: false
        },
    )
    Exercise.associate = function ( models ) {
        Exercise.belongsToMany(models.Program, { through: models.ProgramExercise, allowNull: false });

    }
    return Exercise;
}

File: models/Exercises.cs
namespace FitnessTracker.Models {
    public class Exercise {
        public int Id { get; set; }
        public string Name { get; set; }
        public string Agonist { get; set; }
        public string Description { get; set; }
        public ICollection<TrainingProgram>? TrainingPrograms { get; set; }     

    }
}